bin/.local/bin/tmux-sessionizer

```diff
@@ -1,25 +1,40 @@
 #!/usr/bin/env bash
 
 if [[ $# -eq 1 ]]; then
-    selected=$1
+    # Adding support for simply pass the current working directory or a specified path
+    selected="$1" && [[ "$selected" == '.' ]] && selected="$PWD"
 else
     selected=$(find ~/work/builds ~/ ~/work ~/personal ~/personal/yt -mindepth 1 -maxdepth 1 -type d | fzf)
 fi
 
-if [[ -z $selected ]]; then
-    exit 0
-fi
+# This is work because '&&' do the action only if what came before is true. 
```

Comment:
stylistic changes often can cause slow down in prs.  could we drop this? i am just not going to accept it :)

I like the explicit if statement.

Reply:
Is personal taste. But usually in shell scripting is faster doing a stream instead of a if statement.